airbyte-webapp/src/views/layout/SideBar/AirbyteHomeLink.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { PropsWithChildren } from "react";
+import { useIntl } from "react-intl";
+import { Link } from "react-router-dom";
+
+import { RoutePaths } from "pages/routePaths";
+
+import styles from "./AirbyteHomeLink.module.scss";
+import { ReactComponent as AirbyteLogo } from "./airbyteLogo.svg";
+
+export const AirbyteHomeLink: React.FC<PropsWithChildren<unknown>> = () => {
```

Comment:
nit: this component doesn't actually have children
```suggestion
export const AirbyteHomeLink: React.FC = () => {
```